### ISSUE TITLE
Pressing the escape key when the page is starting up will cause things to break

### ISSUE THREAD

**Issue by uShldGetCeleste** [Miodec/monkeytype#1832]
**Describe the bug** 
If you press the escape key while monkeytype is starting then things will break, usually just visually.

**To Reproduce** 

1. Open the site
2. Before stuff finishes loading press the escape key. The earlier that you press it the more it will be broken

**Expected behavior** 
Page should load properly

**Screenshots** 
![image](https://user-images.githubusercontent.com/68669265/132906885-deb663a3-c592-4bdc-8be2-01571f052e8c.png)
![image](https://user-images.githubusercontent.com/68669265/132906913-9a1d0386-ca4f-48f5-a000-911c81da2528.png)
![image](https://user-images.githubusercontent.com/68669265/132906965-59df6814-e982-4478-a772-829834d62ffa.png)

**Desktop:** 

- OS: Windows 10
- Browser Chrome 
- Browser Version 93.0.4577.63

**Comment by sphhax** [comment]
The escape key in browsers haults all requests so, until the site loads enough to register the escape key as the console button then I am not sure much more can be done. 

The only possible fix would be to put a function at the top of index.html so we can try and load is asap but honestly, this is just normal functionality.

**Comment by rasulkamolov** [comment]
Its been fixed

**Comment by Miodec** [comment]
I keep trying to replicate this and I cant seem to be able to get it.

**Comment by sphhax** [comment]
@Miodec im only able to do it as soon as I refresh the page or by simulating a slower internet connection. The problem likely is that on slower connections it's easier to press escape before the site loads enough to disable it.

**Comment by UnrealApex** [comment]
Try reloading and spamming escape.

**Comment by UnrealApex** [comment]
> @Miodec im only able to do it as soon as I refresh the page or by simulating a slower internet connection. The problem likely is that on slower connections it's easier to press escape before the site loads enough to disable it.

I agree, most computers would be able to load Monkeytype fast enough that these requests can be made before the site is ["interactible"](https://developer.mozilla.org/en-US/docs/Glossary/Time_to_interactive) by the user.

**Comment by UnrealApex** [comment]
Maybe at the top of `index.html` we could add this: 
```html
<script>
document.addEventListener("keydown", function(event) {
  event.preventDefault();
);
</script>
```

**Comment by sphhax** [comment]
Created a PR #1840 in my testing it significantly mitigated this issue, but it's still possible if you haven't loaded the index yet, but it makes it much less likely for this to occur. So It can't hurt, but that is going to be the most we can do.

**Comment by Miodec** [comment]
I dont know how i feel about this fix. I mean, its clearly user error if they press the escape key while the page is loading. Plus if "it's still possible if you haven't loaded the index yet" then whats the point really.

**Comment by sphhax** [comment]
> I dont know how i feel about this fix. I mean, its clearly user error if they press the escape key while the page is loading. Plus if "it's still possible if you haven't loaded the index yet" then whats the point really.

It's not going to matter for the majority of users, it's a band aid to fix an issue for an extremely small amount of users.  Up to you.

**Comment by Miodec** [comment]
Merged #1840
